Core/include/Acts/Seeding/BinnedSPGroup.hpp

```diff
@@ -39,9 +40,12 @@ class BinnedSPGroupIterator {
  public:
   // Never take ownerships
   BinnedSPGroupIterator(BinnedSPGroup<external_spacepoint_t>&& group,
-                        std::size_t) = delete;
+                        std::array<std::size_t, 2> index,
+                        std::array<std::vector<std::size_t>, 2> navigation) =
```

Comment:
totally unrelated.

In this context `navigation` defines a sequence of bins in the axis so that you can jump from one to another when iterating in the grid. This is what we currently do in the seeding for the Z-Axis